CONTRIBUTING.md

```diff
@@ -151,6 +153,13 @@ k8gb project is using the coding style suggested by the Golang community. See th
 Please follow this style to make k8gb easy to review, maintain and develop.
 Run `make check` to automatically check if your code is compliant.
 
+### Logging
+
+k8gb project is using the [zerolog](https://github.com/rs/zerolog) library for logging.
+Please make sure to follow the zerolog library concepts and conventions in the code.
+Try to use [contextual logging](https://github.com/rs/zerolog#contextual-logging) whenever possible.
+Pay attention to [error logging](https://github.com/rs/zerolog#error-logging) recommendations.
```

Comment:
![image](https://user-images.githubusercontent.com/518532/130491421-c525fe11-09ff-46d2-9bdf-d94627531bdc.png)
Probably we need to make a list out of it or empty line in between so it is rendered not as a single line

Reply:
Makes sense!

Reply:
@ytsarev amended